tests/e2e/tests-library/ProjectAndFileTests.ts

```diff
@@ -58,6 +58,33 @@ export class ProjectAndFileTests {
 		}
 	}
 
+	/**
+	 * manage to 'Trusted' Workspace Mode, when the trust dialog does not appear
+	 * @param scmProvider SingleScmProvider object
+	 */
+	async manageWorkspaceTrust(scmProvider: SingleScmProvider): Promise<void> {
```

Comment:
IMHO `manageWorkspaceTrust` method name is too vague.
What about `performManageWorkspaceTrustBox()` method name?

Also it would be nice to say in method comment that it is the box appeared in Source Control View.

Reply:
Fixed.